src/test/crimson/test_cmds.h

```diff
@@ -0,0 +1,76 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
also, could you transpose "crimson" and "test" in the title? so it reads

> test/crimson: extract msgr test commands to header

?

Reply:
sure, fixed.